runner/runner.go

```diff
@@ -927,6 +928,25 @@ retry:
 		fullURL = parsedURL.String()
 	}
 
+	var asnResponse interface{ String() string }
+	if r.options.Asn {
+		addr, err := net.LookupIP(domain)
+		if err != nil {
+			return Result{URL: URL.String(), Input: origInput, err: err}
+		}
+		lookupResult, err := ipisp.LookupIP(context.Background(), addr[0])
+		if err != nil {
+			return Result{URL: URL.String(), Input: origInput, err: err}
```

Comment:
The failed lookup shouldn't be fatal, but print an error warning and leave the variable to nil